testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -327,6 +334,12 @@ else if (getHiveVersionMajor() == 3 && isFormat.test("orc")) {
                 .put("varchar_to_smaller_varchar", Arrays.asList(
                         "ab",
                         "\uD83D\uDCB0\uD83D\uDCB0"))
+                .put("char_to_bigger_char", Arrays.asList(
```

Comment:
nit: if `ImmutableList` is not available then use `List.of()`. `Arrays.asList` is not cool anymore